Simplify DNS records section (#5826)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/manage-hostnames/README.md

<figure><img src="../../../.gitbook/assets/hostnames-umbraco-cloud.png" alt=""><figcaption><p>Manage Hostnames</p></figcaption></figure>

Ensure that the hostname you are binding to your Umbraco Cloud environment has a DNS entry that resolves to the Umbraco Cloud service.

We recommend setting a CNAME record for your hostname. You can also use A & AAAA records for the root of your domain.

* CNAME record value:
  * `dns.umbraco.io`
* A records to either or both IPv4 addresses:
  * `162.159.140.127`
  * `172.66.0.125`
* AAAA records to either or both IPv6 addresses (to support IPv6 connectivity):
  * `2606:4700:7::7d`
  * `2a06:98c1:58::7d`

> If you are using the [Former A and AAAA records](./#former-a-and-aaaa-records) consider changing them to the new A & AAAA records above as they will not become obsolete.

Reviewer: ```suggestion
If you're using the [Former A and AAAA records](./#former-a-and-aaaa-records) consider changing them to the A & AAAA records above as they will not become obsolete.
```

Author: Applied the "shortening" suggestion.
